test/facets/Token/behaviors/bridgeToken.t.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.19;
+
+import { TokenBridge } from "../TokenBridge.t.sol";
+import { ArbForkTest } from "../../../ArbForkTest.t.sol";
+import { ITokenBridge } from "../../../../contracts/facets/Token/ITokenBridge.sol";
+
+/// @title TestBridgeToken
+/// @notice This contract tests the functionalities of the bridgeToken function
+/// @dev It inherits from the TokenWithBridge contract and the ArbForkTest contract
+/// @dev For chaging fork it is sufficient to twick the ForkTest inheritance
+contract TestBridgeToken is ArbForkTest, TokenBridge {
```

Comment:
```suggestion
contract Token_bridgeToken is ArbForkTest, TokenBridge {
```

For all tests please label as: `ContractName_functionName`

Reply:
fixed here https://github.com/giovannifranchi/insrt-v2-contracts/commit/4d1fccbd8f21a673e658c6e216ac59573bb3e306